wd.sh

```diff
@@ -261,12 +261,12 @@ wd_browse() {
     local entries=("${(@f)$(sed "s:${HOME}:~:g" "$WD_CONFIG" | awk -F ':' '{print $1 " -> " $2}')}")
     local script_path="${${(%):-%x}:h}"
     local wd_remove_output=$(mktemp "${TMPDIR:-/tmp}/wd.XXXXXXXXXX")
-    local entries_with_headers=("All warp points:" "Press enter to select. Press delete to remove" "${entries[@]}")
+    entries=("All warp points:" "Press enter to select. Press delete to remove" "${entries[@]}")
     local fzf_bind="delete:execute(echo {} | awk -F ' -> ' '{print \$1}' | xargs -I {} "$script_path/wd.sh" rm {} > "$wd_remove_output")+abort"
-    local fzf_command=$(printf '%s\n' "${entries_with_headers[@]}" | fzf --height 100% --reverse --header-lines=2 --bind="$fzf_bind")
+    local selected_entry=$(printf '%s\n' "${entries[@]}" | fzf --height 100% --reverse --header-lines=2 --bind="$fzf_bind")
     if [[ -e $wd_remove_output ]]; then
         cat "$wd_remove_output"
-        rm "$wd_remove_output"
+        rm "$wd_remove_output" >/dev/null 2>&1
```

Comment:
Why are you expecting `rm` to output?

Reply:
I was getting something about "removing regular file" but then I realized I had rm aliased. I dont seem to be able to reproduce this so Ive removed the pipe

Reply:
I love how thorough you are!